tests/README.tests

```diff
@@ -0,0 +1,6 @@
+To run tests:
+
+* python -m pytest
```

Comment:
If we put the file as a toplevel file. Invoking the tests with `python -m pytest` won't work. Can you put down the command `pytest tests` instead?